sql-statements/sql-statement-create-user.md

```diff
@@ -142,22 +145,38 @@ Create a user whose password is manually expired:
 CREATE USER 'newuser9'@'%' PASSWORD EXPIRE;
 ```
 
+Create a user with a maximum connection limit of 3:
+
+```sql
+CREATE USER 'newuser10'@'%' WITH MAX_USER_CONNECTIONS 3;
+SELECT User, Host, max_user_connections FROM mysql.user WHERE User='newuser10';
+```
+
+```
++-----------+------+----------------------+
+| user      | host | max_user_connections |
++-----------+------+----------------------+
+| newuser10 | %    |                    3 |
++-----------+------+----------------------+
+1 row in set (0.01 sec)
+```
+
 ```
 Query OK, 1 row affected (0.02 sec)
 ```
 
 Create a user that uses the resource group `rg1`.
 
 ```sql
-CREATE USER 'newuser7'@'%' RESOURCE GROUP rg1;
-SELECT USER, HOST, USER_ATTRIBUTES FROM MYSQL.USER WHERE USER='newuser7';
+CREATE USER 'newuser11'@'%' RESOURCE GROUP rg1;
+SELECT USER, HOST, USER_ATTRIBUTES FROM MYSQL.USER WHERE USER='newuser11';
 ```
 
 ```sql
 +----------+------+---------------------------+
 | USER     | HOST | USER_ATTRIBUTES           |
 +----------+------+---------------------------+
-| newuser7 | %    | {"resource_group": "rg1"} |
+| newuser11| %    | {"resource_group": "rg1"} |
 +----------+------+---------------------------+
 1 rows in set (0.00 sec)
 ```
```

Comment:
The following line for `WITH MAX_USER_CONNECTIONS` should be removed according to https://github.com/pingcap/docs-cn/pull/19898/files.
![image](https://github.com/user-attachments/assets/bf57f5d4-b286-4516-a7c2-5caca82899af)

Reply:
Great comments. Ignored by myself~